src/extensions/extensions-store/extensions-store.injectable.ts

```diff
@@ -27,6 +28,7 @@ const extensionsStoreInjectable = getInjectable({
     persistStateToConfig: di.inject(persistStateToConfigInjectionToken),
     enlistMessageChannelListener: di.inject(enlistMessageChannelListenerInjectionToken),
     shouldDisableSyncInListener: di.inject(shouldBaseStoreDisableSyncInIpcListenerInjectionToken),
+    sendMessageToChannel: di.inject(sendMessageToChannelInjectionToken),
```

Comment:
Isn't this already done in the `ExtensionStore` constructor?

Reply:
Haha different type all together. I get messed up too.

There is `ExtensionStore` which is exported to the extension API and then there is `ExtensionsStore` which keeps track of if an extension is enabled or not.

Reply:
oops